extensions/auth/opa/impl/build.gradle.kts

```diff
@@ -58,3 +59,64 @@ dependencies {
   testImplementation(project(":polaris-async-java"))
   testImplementation(project(":polaris-idgen-mocks"))
 }
+
+// Task to generate JSON Schema from model classes
```

Comment:
Couple of Gradle improvements could be applied.
I've created PR against your branch, as it's a bit more: sungwy#3

The idea to have separate generation and validation steps is good IMO, as it makes it obvious that such changes affect the expected OPA contract.

Summary:
* Make both tasks cacheable
* Adjust logging output (keep output brief)
* Moved schema generator class to a separate source-set/configurations, because it's not prod code, but more importantly to not let the json-schema-generator dependency leak into the production runtime classpath